admin-api/central_ledger_api_v1.0.json

```diff
@@ -0,0 +1,3355 @@
+                    },
+                    "406": {
+                        "$ref": "#/components/responses/406"
+                    },
+                    "500": {
+                        "$ref": "#/components/responses/500"
+                    },
+                    "501": {
+                        "$ref": "#/components/responses/501"
+                    },
+                    "503": {
+                        "$ref": "#/components/responses/503"
+                    }
+                }
+            },
+            "put": {
+                "summary": "Update participant accounts",
+                "description": "The PUT /participants/{name}/accounts/{id} request is used to update account details of a participant. Currently, updating only the isActive flag is supported.",
+                "operationId": "putParticipantsNameAccountsId",
+                "parameters": [
+                    {
+                        "$ref": "#/components/parameters/ParticipantName"
+                    },
+                    {
+                        "$ref": "#/components/parameters/AccountIdentifier"
+                    }
+                ],
+                "requestBody": {
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "$ref": "#/components/schemas/AccountIsActive"
+                            }
+                        }
+                    }
+                },
+                "tags": [
+                    "participants"
+                ],
+                "responses": {
+                    "200": {
+                        "$ref": "#/components/responses/200"
+                    },
+                    "400": {
+                        "$ref": "#/components/responses/400"
+                    },
+                    "401": {
+                        "$ref": "#/components/responses/401"
+                    },
+                    "403": {
+                        "$ref": "#/components/responses/403"
+                    },
+                    "404": {
+                        "$ref": "#/components/responses/404"
+                    },
+                    "405": {
+                        "$ref": "#/components/responses/405"
+                    },
+                    "406": {
+                        "$ref": "#/components/responses/406"
+                    },
+                    "500": {
+                        "$ref": "#/components/responses/500"
+                    },
+                    "501": {
+                        "$ref": "#/components/responses/501"
+                    },
+                    "503": {
+                        "$ref": "#/components/responses/503"
+                    }
+                }
+            }
+        },
+        "/participants/{name}/accounts/{id}/transfers/{transferId}": {
+            "put": {
+                "summary": "Record Funds In or Out of participant account",

```

Comment:
@elnyry-sam-k I think this description should be made more specific, otherwise we have two end-points with the exact same description.

I was thinking something like this: "Record a Transfer as a Funds In or Out transaction for a participant account"